lib/src/screens/initial_setup/widgets/initial_setup_welcome.dart

```diff
@@ -94,80 +84,86 @@ SizedBox initialSetupWelcome(BuildContext context) {
-        style: TextStyle(
-          color: Colors.grey[800],
-          letterSpacing: 2.0,
-          fontSize: 17.0,
-          fontWeight: FontWeight.bold,
-          fontFamily: 'Poppins',
-        ),
-      ),
-    ],
-  );
-}
+// /// Creates a row widget displaying a piece of profile information.
+// ///
+// /// This function constructs a `Row` widget designed to display a single piece
+// /// of information in a profile UI. It is primarily used for laying out text-based
+// /// information such as names, titles, or other key details in the profile section.
 
-/// Builds a row widget displaying a label and its corresponding value.
-///
-/// This function creates a [Column] widget containing a [Row] with two text elements:
-/// one for the label and the other for the profile name. It's used to display
-/// information in a key-value pair format, where `labelName` is the key and
-/// `profName` is the value.
+// Row buildInfoRow(String profName) {
+//   return Row(
+//     mainAxisAlignment: MainAxisAlignment.spaceBetween,
+//     children: <Widget>[
+//       Text(
+//         profName,
+//         style: TextStyle(
+//           color: Colors.grey[800],
+//           letterSpacing: 2.0,
+//           fontSize: 17.0,
+//           fontWeight: FontWeight.bold,
+//           fontFamily: 'Poppins',
+//         ),
+//       ),
+//     ],
+//   );
+// }
 
-Column buildLabelRow(String labelName, String profName, BuildContext context) {
-  return Column(
-    children: [
-      Row(
-        mainAxisAlignment: MainAxisAlignment.spaceBetween,
-        children: <Widget>[
-          Text(
-            '$labelName: ',
-            style: TextStyle(
-              color: kTitleTextColor,
-              letterSpacing: 2.0,
-              fontSize: screenWidth(context) * 0.015,
-              fontWeight: FontWeight.bold,
-              //fontFamily: 'Poppins',
-            ),
-          ),
-          profName.length > longStrLength
-              ? Tooltip(
-                  message: profName,
-                  height: 30,
-                  textStyle: const TextStyle(fontSize: 15, color: Colors.white),
-                  verticalOffset: kDefaultPadding / 2,
-                  child: Text(
-                    truncateString(profName),
-                    style: TextStyle(
-                      color: Colors.grey[800],
-                      letterSpacing: 2.0,
-                      fontSize: screenWidth(context) * 0.015,
-                    ),
-                  ),
-                )
-              : Text(
-                  profName,
-                  style: TextStyle(
-                      color: Colors.grey[800],
-                      letterSpacing: 2.0,
-                      fontSize: screenWidth(context) * 0.015),
-                ),
-        ],
-      ),
-      SizedBox(
-        height: screenHeight(context) * 0.005,
-      )
-    ],
-  );
-}
+// comment out the following function as it is not used in the current version
```

Comment:
Hi @anushkavidanage , could you please have a look at the following buildLabelRow widget , if it is not needed, I can delete them. If we need to use them in the future, we can comment out just like what I did. :)